website/docs/monitors/0400-lspci.md

```diff
@@ -0,0 +1,52 @@
+﻿# lspci
+The PCI Utilities package contains a library for portable access to PCI bus
+configuration registers and several utilities based on this library.
+
+- lspci: displays detailed information about all PCI buses and devices.
+
+* [lspci repo](https://github.com/pciutils/pciutils)
+* [Windows Build Download](https://eternallybored.org/misc/pciutils/)
+
+## Dependency
+Most Linux distro comes with lspci pre-installed. On Windows, VC is packaging a win-x64 build inside VC package itself.
+
+## Supported Platforms
+* linux-x64
+* linux-arm64
+* win-x64
+
+
+## lspci Output Description
+The following section describes the various counters/metrics that are available with the lspci toolset.
+
+The command we are using is `lspci -vvv`. The data structure of the lspci output is not a straightforward dicctionary. 
+The data structure is parsed according to the contract at [PciDevice.cs](../../../src/VirtualClient/VirtualClient.Monitors/Lspci/PciDevice.cs).
+
+For each PCI device, we are parsing the name, address, various properties and capabilities.
+
+### Example
```

Comment:
Update the documentation for the monitor profiles:
https://microsoft.github.io/VirtualClient/docs/monitors/0200-monitor-profiles/

This is where I have been documenting the differences for each one of our MONITORS-* profiles.